### PR TITLE
Add mempool v1 docker image using `opt-add-mempool` fragment

### PR DESCRIPTION
**Pull request by wiz**
Blocked by https://github.com/mempool/mempool/issues/17

### CODE REVIEW

File: Production/nginx.tmpl [hunk 1]
```diff
@@ -32,6 +32,18 @@
...
 		rewrite ^/lnd-rest/btc/(.*) /$1 break;
 		proxy_pass http://lnd_bitcoin:8080/;
+	}
+		{{ end }}
+		{{ if (eq $serviceName "bitcoin_mempool") }}
+	location /mempool {
+		rewrite ^/mempool(.*)$ $1 break;
+		proxy_pass http://bitcoin_mempool/;
+	}
+	location /ws {
```
Comment (Kukks) [inline Production/nginx.tmpl:42]: too vague, can't it be `mempool/ws`?

**Review by Kukks — CHANGES_REQUESTED** [review]
Can you also add
* readme docs
* Docker build instructions to https://github.com/btcpayserver/btcpayserver-docker/blob/7fda5c2564f0bf4ed33a665523ed11340bcc72ab/contrib/DockerFileBuildHelper/Program.cs#L556
* Run https://github.com/btcpayserver/btcpayserver-docker/blob/7fda5c2564f0bf4ed33a665523ed11340bcc72ab/contrib/DockerFileBuildHelper/update-repo.sh after

**Comment by NicolasDorier** [comment]
@wiz planning to do?

**Comment by wiz** [comment]
Yeah, sorry for the delay but instead of shipping v1 for btc we just shipped v2 with core RPC backend support so I can update this PR next
https://github.com/mempool/mempool/releases/tag/v2.0.0

**Comment by Eskyee** [comment]
thanks @wiz 
awesome stuff

**Review by uhlhosting — COMMENTED** [review]
I tried this and I got:

`process caused: exec: "/mempool.space/entrypoint.sh": stat /mempool.space/entrypoint.sh: no such file or directory: unknown`

Any ideas?

**Comment by wiz** [comment]
This PR isn't ready for testing yet. The latest update is that we shipped v2 with dockerfile support and also got it integrated into RaspiBlitz, Umbrel, MyNode, and soon RoninDojo.. btcpayserver-docker is next!

**Comment by maltokyo** [comment]
@wiz @NicolasDorier I got this working perfectly using my own custom docker fragment inside BTCPay server.

Here it is (of course the redacted parts need taking care of, and adding it as a tor service, adding the env vars etc needs doing, but I can confirm mine works brilliantly!) - please use as needed:

```yml
version: "3.7"

x-logging:
  &default-logging
  driver: journald


services:
  mempool_web:
    container_name: mempool_web
    image: mempool/frontend:v2.1.2@sha256:69fefe55dc1eb4c8373c32d57362df7eeb672fc591d55033d68ae79355274dcc
    init: true
    logging: *default-logging
    restart: on-failure
    stop_grace_period: 1m
    command: "./wait-for mempool_db:3306 --timeout=720 -- nginx -g 'daemon off;'"
    # ports:
    #   - 3006:3006
    environment:
      FRONTEND_HTTP_PORT: "80"
      BACKEND_MAINNET_HTTP_HOST: "mempool_api"

  mempool_api:
    container_name: mempool_api
    image: mempool/backend:v2.1.2@sha256:58394cb35aad82b95ab2496fd4a6d3c983a712a1003454d94b0a53197d306f6d
    init: true
    logging: *default-logging
    restart: on-failure
    stop_grace_period: 1m
    command: "./wait-for-it.sh mempool_db:3306 --timeout=720 --strict -- ./start.sh"
    volumes:
      - mempoolapidir:/backend/cache
    environment:
      RPC_HOST: "bitcoind"
      RPC_PORT: "43782"
      RPC_USER: "REDACTED"
      RPC_PASS: "REDACTED"
      ELECTRS_HOST: "electrumx"
      ELECTRS_PORT: "50002"
      MYSQL_HOST: "mempool_db"
      MYSQL_DATABASE: "mempool"
      MYSQL_USER: "mempool"
      MYSQL_PASS: "REDACTED"
      BACKEND_MAINNET_HTTP_PORT: "8999"
      CACHE_DIR: "/backend/cache/"
      MEMPOOL_CLEAR_PROTECTION_MINUTES: "20"

  mempool_db:
    container_name: mempool_db
    image: mariadb:10.5.8@sha256:8040983db146f729749081c6b216a19d52e0973134e2e34c0b4fd87f48bc15b0
    logging: *default-logging
    restart: on-failure
    stop_grace_period: 1m
    volumes:
      - mempooldbdir:/var/lib/mysql
      - mempooldbscriptsdir:/docker-entrypoint-initdb.d
    environment:
      MYSQL_DATABASE: "mempool"
      MYSQL_USER: "mempool"
      MYSQL_PASSWORD: "REDACTED"
      MYSQL_ROOT_PASSWORD: "REDACTED"

# use docker native volumes
volumes:
  mempoolapidir:
  mempooldbdir:
  mempooldbscriptsdir:
```

**Comment by wiz** [comment]
Thanks for working on this, will close this PR and look forward to review yours 🚀

**Pull request closed without merging**